x-pack/plugins/timelines/public/container/use_update_alerts.ts

```diff
@@ -23,17 +24,20 @@ export const DETECTION_ENGINE_SIGNALS_STATUS_URL = '/api/detection_engine/signal
  */
 export const useUpdateAlertsStatus = (): {
   updateAlertStatus: (params: {
-    query: object;
     status: AlertStatus;
+    index: string;
+    ids?: string[];
+    query?: object;
   }) => Promise<estypes.UpdateByQueryResponse>;
 } => {
   const { http } = useKibana().services;
-
   return {
-    updateAlertStatus: ({ query, status }) =>
-      http!.fetch(DETECTION_ENGINE_SIGNALS_STATUS_URL, {
+    updateAlertStatus: async ({ status, index, ids, query }) => {
+      const { body } = await http!.fetch(RAC_ALERTS_BULK_UPDATE_URL, {
```

Comment:
nit: `http!.fetch`, the bang shouldn't be needed here. Don't know when it ended up here, but looking across the code base this bang doesn't look required.